src/ansys/additive/core/parametric_study/parametric_study.py

```diff
@@ -746,59 +772,84 @@ def generate_microstructure_permutations(
-            Minimum y coordinate (m) of the microstructure sample.
+            Minimum y coordinate (m) of the microstructure sample. Valid values are from
+            :obj:`MIN_POSITION_COORDINATE <MicrostructureInput.MIN_POSITION_COORDINATE>`
+            to :obj:`MAX_POSITION_COORDINATE <MicrostructureInput.MAX_POSITION_COORDINATE>`.
         min_z : float, default: :obj:`DEFAULT_POSITION_COORDINATE <MicrostructureInput.DEFAULT_POSITION_COORDINATE>`
-            Minimum z coordinate (m) of the microstructure sample.
+            Minimum z coordinate (m) of the microstructure sample. Valid values are from
+            :obj:`MIN_POSITION_COORDINATE <MicrostructureInput.MIN_POSITION_COORDINATE>`
+            to :obj:`MAX_POSITION_COORDINATE <MicrostructureInput.MAX_POSITION_COORDINATE>`.
         size_x : float, default: :obj:`DEFAULT_SAMPLE_SIZE <MicrostructureInput.DEFAULT_SAMPLE_SIZE>`
             Size (m) of the microstructure sample in the x direction.
-            Valid values are between 0.001 and 0.01.
+            Valid values are from :obj:`MIN_SAMPLE_SIZE <MicrostructureInput.MIN_SAMPLE_SIZE>`
+            to :obj:`MAX_SAMPLE_SIZE <MicrostructureInput.MAX_SAMPLE_SIZE>`.
         size_y : float, default: :obj:`DEFAULT_SAMPLE_SIZE <MicrostructureInput.DEFAULT_SAMPLE_SIZE>`
             Size (m) of the microstructure sample in the y direction.
-            Valid values are between 0.001 and 0.01.
+            Valid values are from :obj:`MIN_SAMPLE_SIZE <MicrostructureInput.MIN_SAMPLE_SIZE>`
+            to :obj:`MAX_SAMPLE_SIZE <MicrostructureInput.MAX_SAMPLE_SIZE>`.
         size_z : float, default: :obj:`DEFAULT_SAMPLE_SIZE <MicrostructureInput.DEFAULT_SAMPLE_SIZE>`
             Size (m) of the microstructure sample in the z direction.
-            Valid values are between 0.001 and 0.01.
+            Valid values are from :obj:`MIN_SAMPLE_SIZE <MicrostructureInput.MIN_SAMPLE_SIZE>`
+            to :obj:`MAX_SAMPLE_SIZE <MicrostructureInput.MAX_SAMPLE_SIZE>`.
         sensor_dimension : float, default: :obj:`DEFAULT_SENSOR_DIMENSION <MicrostructureInput.DEFAULT_SENSOR_DIMENSION>`
             Sensor dimension (m) to use for microstructure simulations.
-            Valid values are between 0.0001 and 0.001. The values for the
-            ``size_x`` and ``size_y`` parameters must be greater than the
-            ``sensor_dimension`` parameter by 0.0005. The value for the
-            ``size_z`` parameter must be greater than ``sensor_dimension``
-            parameter by 0.001.
+            Valid values are from :obj:`MIN_SENSOR_DIMENSION <MicrostructureInput.MIN_SENSOR_DIMENSION>`
+            to :obj:`MAX_SENSOR_DIMENSION <MicrostructureInput.MAX_SENSOR_DIMENSION>`.
+            The values for the ``size_x`` and ``size_y`` parameters must be greater than the
+            ``sensor_dimension`` parameter by :obj:`MIN_XY_SIZE_CUSHION <MicrostructureInput.MIN_XY_SIZE_CUSHION>`.
```

Comment:
Would adding a unit be better here: by :obj:`MIN_XY_SIZE_CUSHION <MicrostructureInput.MIN_XY_SIZE_CUSHION>` meters. It seems a bit awkward as is.

Reply:
Done.